### PR TITLE
Fix "My Institution" Links

### PR DESCRIPTION
**Pull request by thostetler**
- Add the "My Institution" link to abstract page
- Fix the link server settings page
- Some fixes for linkGenerator

### CODE REVIEW

**Comment by shinyichen** [comment]
I've not finish looking at the code, but noticed a couple of things.
* There is no 'clear' for the library link server, maybe it makes sense to add it with the PR.
* After logging out, I still see 'My institution' link in the full text resources. Have to do a refresh to clear it.

**Comment by thostetler** [comment]
@shinyichen thanks.  I did add a clear button now, good idea! 

I will fix the logout issue with another PR.  It's a larger fix.

**Review by shinyichen — APPROVED** [review]
Just the comment on icon color, otherwise looking good.

File: src/components/AbstractSources/AbstractSources.tsx [hunk 1]
```diff
@@ -78,23 +74,37 @@ interface IFullTextDropdownProps {
...
-        <LockIcon mr={1} />
+        <UnlockIcon color="green.600" mr={1} />
         {` ${source.name}`}
       </>
-    ),
+    );
+  }
+  return (
+    <>
+      <LockIcon color="gray.700" mr={1} />
```
[the remark this answers is not in the document]
Comment (thostetler) [reply src/components/AbstractSources/AbstractSources.tsx]: :+1:

**Pull request merged**